ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+What would be the second thing that they would want to do?
+And now that they've managed to complete that what is the next thing?
+
+When you've got tasks for all of your readers, let's put it all together.
+
+## Writing for your readers
+
+Now that you know who your readers are, and the kinds of things they might be using your documentation for, you can display them in a table like this:
+
+<table>
+    <tr>
+        <th>Tasks</th>
+        <th>Reader 1</th>
+        <th>Reader 2</th>
+        <th>Reader 3</th>
+        <th>Critical Path Score</th>
+    </tr>
+    <tr>
+        <td>Task 1</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 2</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 3</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 4</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 5</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 6</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 7</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 8</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>Task 9</td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+</table>
+
+Now you just need to fill in the blank spaces.
+Consider how likely the reader is to use the documentation to complete each task.
+Score a **1** for a low likelihood, **2** for a medium likelihood, and **3** for a high likelihood.
+
+**Curse of Knowledge alert!!**
+> Remember that this isn't how likely the reader is to do the task.
+> It's how likely the reader is to *use the documentation* to do the task.
+> A really experienced reader is not likely to need to the documentation for tasks like installation or setup, but they might need it for more advanced tasks.
+
+When you have the numbers in, add up each row.
+The rows that score the highest are what writers like to call *critical paths*.
+The items with the highest critical path score is the content that is most important for you to write.
+
+## Identify the mess
+
+It's time to turn our attention to what you already have.
+If you have already made a start on the docs, that's great!
+But even if you think you're starting from scratch, there's a reasonable chance you actually already have something.
+
+Have a think about it:
+
+* Have you written notes while you've been working?
+    Even if they're just scratches on a notebook on your desk, that's documentation!
+* Have you written any comments in your code?
```

Comment:
other helpful places to look are Pull-Request and Issue-Trackers on github, those are documentation as well